### PR TITLE
[New Vault] Nanotrasen Experimental AME Lab

### PR DESCRIPTION
**Pull request by DeityLink**
image

To not be merged before #30036

:cl:
* experiment: A satellite has been observed drifting through Nanotrasen space which is believed to be the testing grounds of a new experimental type of AME. Whatever tests were done there appear to have stopped and the satellite was evacuated in a hurry. The satellite itself was deemed too unsafe to properly decommission so it is still drifting through space to this day. Regardless, do not go near it. Do not board it. And by all means Do Not try to resume the experiments that were happening there.

### CODE REVIEW

**Comment by Kurfursten** [comment]
Nice. But maybe it could use a portable SMES, since the researchers would probably want to be collecting power?

**Comment by DeityLink** [comment]
> 
> 
> Nice. But maybe it could use a portable SMES, since the researchers would probably want to be collecting power?

It's less about production and more about measurement, and now that you mention it I should definitely add a power monitoring console.

**Comment by Eneocho** [comment]
You could add a renamed/reskinned emitter that something something sends power to a distribution facility via bluespace pulses for fluff and immulsion purposes.

File: maps/randomvaults/objects.dm [hunk 1]
```diff
@@ -198,6 +198,70 @@
...
+
+/obj/item/weapon/paper/amelab4
+	name = "paper- 'Fourth Report'"
+	info = "Centcomm has shipped me some much larger fuel jars in preparation for Phase Two. A scientist will be coming, an Anomalist specifically. I'm not quite versed into Exotic Particle science but they will test alternative ways to raise the power output. This can go wrong in so many ways but we're all well aware of the risks, and the pay is good enough. Worst case scenario the lab explodes and we'll be the only casualties."
+
+/obj/item/weapon/paper/amelab5
+	name = "paper- 'Fifth Report'"
+	info = "The anomalist has started their experiments on the engine, they brought various devices and machines I do not recognize. Regardless, their radiations their tools emit seem to have a positive effect on the power output. Looks like we're on the verge of a technological breakthrough! With the last deliveries we also ordered a portable SMES kit. If we can export enough surplus energy we might not only earn ourselves a bonus, but also prove the commercial applications of the enhanced AME."
```
Comment (Eneocho) [inline maps/randomvaults/objects.dm]: The "their radiations their tools emit" part on the 5th report feels off.

Comment (DeityLink) [reply maps/randomvaults/objects.dm]: Yep that's a typo

```suggestion
	info = "The anomalist has started their experiments on the engine, they brought various devices and machines I do not recognize. Regardless, the radiations their tools emit seem to have a positive effect on the power output. Looks like we're on the verge of a technological breakthrough! With the last deliveries we also ordered a portable SMES kit. If we can export enough surplus energy we might not only earn ourselves a bonus, but also prove the commercial applications of the enhanced AME."
```

**Comment by DeityLink** [comment]
I see you coming, no emitters in this Vault. I already added a portable SMES, the pic isn't completely up to date anymore but that's fine.

**Comment by SonixApache** [comment]
shame we don't have anything that burns away power outside of the ancient matter synth and starscreens, would be fitting here

**Comment by kernikov** [comment]
Vault isn't listed as a map element in the "load map element" button.

**Pull request merged**